posthog/temporal/tests/batch_exports/test_spmc.py

```diff
@@ -146,7 +146,7 @@ def test_slice_record_batch_into_single_record_slices():
     """Test we slice a record batch into slices with a single record."""
     n_legs = pa.array([2, 2, 4, 4, 5, 100])
     animals = pa.array(["Flamingo", "Parrot", "Dog", "Horse", "Brittle stars", "Centipede"])
-    batch = pa.RecordBatch.from_arrays([n_legs, animals], names=["n_legs", "animals"])
+    batch = pa.RecordBatch.from_arrays([n_legs, animals], names=["n_legs", "animals"])  # type: ignore
```

Comment:
Same on these ones, looks like we should just be able to type hint this correctly?